.github/workflows/test-tutorials.yml

```diff
@@ -2,18 +2,38 @@ name: Test-Tutorials
 
 on:
   push:
+    branches:
+      - main
```

Comment:
```suggestion
on:
  push:
    branches:
      - main
  workflow_dispatch: {}
```

Reply:
There was already a `name: Test-Tutorials` in the first line. I'll remove the one added with this suggestion. Does it work for you?

Reply:
Also, run workflow doesn't show up in Actions. Are you able to run it?

Reply:
These changes have to be merged into the `main` first to let the workflow be manually triggered.

Reply:
If you want to make this workflow to execute on every push and PR then it should be something like:

```yaml
on:
  push: {}
  pull_request: {}
  workflow_dispatch: {}
```

Reply:
I don’t think it should be executed on every push and PR, as changes in doc2 won’t affect how examples run. Running it on every push to main is sufficient, IMO.